repositories/vaccineOps.js

```diff
@@ -0,0 +1,20 @@
+const axios = require("axios");
+
+module.exports = {
+  getUsData: async (state) => {
```

Comment:
nitpick: I would probably update the name of this function to better describe what it's doing.  we aren't getting vaccine data for the entire U.S, we're only doing it for a specific state.  so perhaps something like `getUsDataByState`